src/views/Game/GameDock.tsx

```diff
@@ -325,123 +329,169 @@ export function GameDock({
                     {pgettext("Control who can access the game or review", "Access settings")}
                 </a>
             )}
+            <Tooltip tooltipRequired={tooltipRequired} title={"Toggle Volume"}>
+                <a>
+                    <i
+                        className={
+                            "fa volume-icon " +
+                            (volume === 0
+                                ? "fa-volume-off"
+                                : volume > 0.5
+                                ? "fa-volume-up"
+                                : "fa-volume-down")
+                        }
+                        onClick={toggleVolume}
+                    />{" "}
+                    <input
+                        type="range"
+                        className="volume-slider"
+                        onChange={setVolume}
+                        value={volume}
+                        min={0}
+                        max={1.0}
+                        step={0.01}
+                    />
+                </a>
+            </Tooltip>
+            <Tooltip
+                tooltipRequired={tooltipRequired}
+                title={"Zen Mode"}
```

Comment:
Thanks for the PR, looks good! We do need to get those translated though, but that's fairly easy. To handle translations we wrap it with `_("text")` or `pgettext("some translation context", "text")`, these translate the strings into whatever language the user is using.

Reply:
Not a problem! Is there a difference/preference between the two options?

Reply:
The difference is that `pgettext` provides some context for translators. For instance "rank" might be translated differently depending on whether we're talking about a ladder rank, or your playing ability, so you would provide some additional string context to both differentiate the string for translation and provide that string to translators to let them know how it will be used.

If that extra context isn't needed though, just use `_`.

In this case you can probably usually use whatever was used for the string that's visible when the dock slides out. If you use the same string and same call, it will use existing translations which is preferable.

Reply:
![image](https://github.com/online-go/online-go.com/assets/76088738/90d563b7-bd6c-45c2-aba5-c71f4749ae4a)

I have noticed that some of the strings in the dock currently do not have anything for translation (Fast AI Review for example). Have these been left out intentionally? If not, happy to add them now as well

Reply:
Yeah, those are things restricted for super users, so I don't include them there so the translators aren't asked to translate stuff that's not visible to the general population